external/loaders/loaders/mappers/__init__.py

```diff
@@ -7,7 +7,7 @@
 )
 from ._transformations import ValMap, KeyMap, SubsetTimes
 from ._local import LocalMapper, mapper_to_local
-from ._hybrid import open_fine_resolution_nudging_hybrid
+from ._hybrid import open_fine_resolution_nudging_hybrid, open_fine_resolution
```

Comment:
nit: generally seems like a duplication of code to do all this renaming in multiple `__init__.py`. Makes it hard to follow what is defined where.